### PR TITLE
Rename EEC to Extreme Entity Crusher

### PR DESCRIPTION
**Pull request by Alexdoru**
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/14953

### CODE REVIEW

File: src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java [hunk 1]
```diff
@@ -259,7 +259,7 @@ protected IAlignmentLimits getInitialAlignmentLimits() {
     protected GT_Multiblock_Tooltip_Builder createTooltip() {
         GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
         tt.addMachineType("Powered Spawner")
-            .addInfo("Controller block for the Extreme Entity Crusher")
+            .addInfo("Controller block for the EEC : Extreme Entity Crusher")
```
Comment (kuba6000) [inline src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: EEC should be written in a separated line like in mega apiary (`AKA. EEC`)

**Review by kuba6000 — CHANGES_REQUESTED** [review]

File: src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java (hunk 1 above)
Comment (Alexdoru) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: btw shouldn't the aka be lowercase ?

**Comment by Alexdoru** [comment]
I think it looks better and less clumsy on the same line and also when used it's `aka` and not `AKA.`

File: src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java [hunk 2]
```diff
@@ -259,7 +259,7 @@ protected IAlignmentLimits getInitialAlignmentLimits() {
     protected GT_Multiblock_Tooltip_Builder createTooltip() {
         GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
         tt.addMachineType("Powered Spawner")
-            .addInfo("Controller block for the EEC : Extreme Entity Crusher")
+            .addInfo("Controller block for the Extreme Entity Crusher aka EEC")
```
Comment (kuba6000) [inline src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: I think this line should be clean with the full name only, so revert your last commit and make it in separated line like it was in MApiary.

**Review by kuba6000 — CHANGES_REQUESTED** [review]

File: src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java (hunk 2 above)
Comment (Alexdoru) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: I think it looks better if it's on the same line tho 
image
image

Comment (kuba6000) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: `Aka Mega Apiary` should be under `The ideal home...`
`Aka EEC` should be under `Spawns and kills...`

Comment (Alexdoru) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: that's not how you use `aka` in english, you are supposed to say `full name aka alias`, so it should be after the full name and not on another separate line

Comment (kuba6000) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: It's close enough and looks much better down there imo.
It shouldn't be in the `Controller block for...` line for sure.

Comment (Alexdoru) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: well I think the opposite so what do we do

Comment (bombcar) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: The Extreme Entity Crusher (EEC)

**Comment by Dream-Master** [comment]
@kuba6000

File: src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java (hunk 2 above)
Comment (kuba6000) [reply src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeEntityCrusher.java]: This is still not resolved, as I said this line should only cointain full name of the multiblock, also `Mega Apiary` goes out of screen on auto GUI scale now
I suggest to change Machine type in the tooltips
`Machine type: Mega Apiary`
`Machine type: Powered Spawner, EEC`
if you dont want to add it lower as I said earlier

**Review by kuba6000 — APPROVED** [review]

**Pull request merged**